source/Ship.cpp

```diff
@@ -4969,3 +4969,10 @@ double Ship::CalculateDeterrence() const
 		}
 	return tempDeterrence;
 }
+
+
+
+bool Ship::Immitates(const Ship &other) const
+{
+	return displayModelName == other.DisplayModelName() && outfits == other.Outfits();
+}
```

Comment:
The order of functions in the cpp file should match the order of functions in the header file. So this should be between Linger and StepFlags, not all the way at the bottom of the file.